### ISSUE TITLE
Remove Redownload skill tree assets?

### ISSUE THREAD

**Issue by SpaceOgre** [EmmittJ/PoESkillTree#441]
This seems to only cause confusion and create problems so maybe it should be removed? Or it might need an information window before use?

Another way could be to make our own repo of assets that the program can pull from.

**Comment by EmmittJ** [comment]
It used to have an information window (I can't check) that said exactly what it does before starting. It is helpful for those that want to keep an old version of the program.

**Comment by SpaceOgre** [comment]
It have this information atm:
![image](https://cloud.githubusercontent.com/assets/381657/23512438/dfa38a90-ff60-11e6-9f73-0c131fb1ea36.png)

I agree the feature it self is good to keep for that reason. Maybe add extra information to the popup that the assets are downloaded from the official tree and that you get that version. I guess it is mostly a problem for a day or two when we have new data before the official tree is updated together with the game.

**Comment by EmmittJ** [comment]
A bit more info would be good, but people don't read. We many want to
change that to a warning sign.

**Comment by brather1ng** [comment]
In addition to a sentence more in the popup, maybe the menu entry can be worded better? At least the menu entry should be read by everyone,

**Comment by EmmittJ** [comment]
Something along the lines of "Get Live Skill Tree Data" would do. We would have to make it clear that Live means from the GGG website and not packaged data.

**Comment by JonathanShor** [comment]
There seems to be an issue with that feature right now. Running version 2.6.0 of the app, after selecting it and seeing the download progress bar fill, it then displays the following:

![image](https://cloud.githubusercontent.com/assets/11262246/24068488/6acd598c-0b66-11e7-8c38-e1a7df2ed2c9.png)

**Comment by brather1ng** [comment]
How about "Redownload Live Skill Tree Data" as menu entry and

![redownload](https://cloud.githubusercontent.com/assets/7596346/25066081/6385a1b2-221c-11e7-931d-f6a660767947.PNG)

Regarding the crash above: I could reproduce that once and after that it was fine. From what I saw in the exception, it seems a key of a dictionary is not in the dictionary anymore a few lines later with nothing modifying the dictionary in between, so that is weird. (if someone else wants to try to fix it: the exception is in SkillTree, line 372, `ints[0]` is not in `Skillnodes`)

**Comment by JonathanShor** [comment]
I think the crash I reported involves some combination of #439 and the auto-upgrade process.
While I was running 2.6.0, it had been upgraded in place over (I believe) 2.3. Digging a bit had turned up the same "No-pseudo-" etc line in debuq.txt.

However, following https://github.com/EmmittJ/PoESkillTree/wiki/Program-crash:-%22No-Pseudo-Attributes-loaded.%22 did not solve the problem. I recall attempting to reload/reset as many assets and user files as I could find instructions for, but the problem persisted.

Fresh download & install ultimately resolved. Manually moving a few builds over to the new install caused no issues either.

**Comment by brather1ng** [comment]
Only thing I gathered from the one time I could reproduce it was that it should be impossible to happen (which it obviously isn't). With that there isn't much I can do to fix it. But you could resolve it and it doesn't seem to be a frequent issue, so that's good.

Anyway, thanks for reporting it. If it does occur again, please just open a new issue.
